synapse/cortex.py

```diff
@@ -2788,6 +2788,13 @@ async def _normStormPkg(self, pkgdef, validstorm=True):
                    f'Cortex is running {s_version.version}'
             raise s_exc.BadVersion(mesg=mesg)
 
+        # Check synapse version requirement
+        reqversion = pkgdef.get('synapse_requirement')
```

Comment:
as usual, i wanna discuss the names etc ;)

Reply:
updated to `synapse_version`